### ISSUE TITLE
Proxy

### ISSUE THREAD

**Issue by WeAreLiving1984** [breezy-weather/breezy-weather#1426]
### Steps to reproduce

Open-meteo is blocked by my ISP it seems, it's great weather source I don't want to ditch it.

Can we have support for socks5 proxy with auth? That would solve my issue.

For now I guess I will have to add breezy weather to my vpn white list as workaround.

### Expected behavior

Weather updating without errors

### Actual behavior

1 update(s) failed

I also have checked on laptop on same network and I can access open-meteo there ether.

### Weather source used

Open-Meteo

### Breezy Weather version

5.2.8

### Android version

Android 13

### Device

Samsung

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
You can find proxy settings in Android settings, not in Breezy Weather.

Additionally, having Open-Meteo blocked by your ISP (or VPN?) is not a Breezy Weather-related bug.

Side note: a VPN/proxy is shared by many users, so you may encounter a different issue with rate-limits (Open-Meteo is limited to 10000 calls per IP per day).

**Comment by papjul** [comment]
> I didn't know, can I use proxy only for Breezy Weather this way?

No, but maybe check projects like RethinkDNS

> I believe it's almost every ISP in a country. Guessing you don't have many users from this country.

What I don't understand is that you say if you whitelist from your VPN, it works. If you whitelist, it means you no longer go through your VPN but through your ISP. So it means it works with your ISP, but not your VPN, isn't it?

**Comment by papjul** [comment]
Is it Russia? I would be surprised no one mentioned it before, if that's the case.

Can you try going to `about:config` and setting `security.tls.version.max` to `3` to confirm it's related to TLS 1.3?

**Comment by papjul** [comment]
Maybe they make it work in the morning for business needs (because a lot of websites must be broken otherwise), and then shutdown again?

Since the issue will automatically closed, if you encounter the issue again, or have tips to share, please use the Q&A section: https://github.com/breezy-weather/breezy-weather/discussions/categories/q-a
